Assignment7/src/main/java/com/assignment/springmaven/controllers/EmployeeController.java

```diff
@@ -0,0 +1,53 @@
+package com.assignment.springmaven.controllers;
+
+import com.assignment.springmaven.model.Employee;
+import com.assignment.springmaven.service.EmployeeServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+import java.util.List;
+
+@Controller
+public class EmployeeController {
+
+    @Autowired
+    private EmployeeServiceImpl employeeService;
+
+    @RequestMapping(value = "/addEmployee")
```

Comment:
Extract @RequestMapping(value = "/employee") at the top of the class and use it in the class so that there isn't any need to mention this mapping everywhere.